source/conf.py

```diff
@@ -21,8 +21,7 @@
 master_doc = "index"
 
 project = 'Kokkos'
-# copyright = f'{datetime.datetime.now().year}, 2021, National Technology & Engineering Solutions of Sandia, LLC (NTESS)'
-copyright = u"2014, National Technology & Engineering Solutions of Sandia, LLC (NTESS)"
+copyright = f'{datetime.datetime.now().year}, National Technology & Engineering Solutions of Sandia, LLC (NTESS)'
```

Comment:
this has to modified because the copyright does not change anytime we build this. We need to fix this to whatever year is listed in the headers inside the code

Reply:
ok, changing to 2014 as it was. I thought it needs to be up-to-date with current year, when it was created and deployed